src/index.js

```diff
@@ -1,8 +1,14 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import registerServiceWorker from './registerServiceWorker';
+import { createClient } from 'service-mocker/client'
```

Comment:
place react import as first in imports order. Let's keep a consistency among files